source/API/CMakeLists.txt

```diff
@@ -170,7 +170,16 @@ if(LLDB_BUILD_FRAMEWORK)
 endif()
 
 set(lib_dir "${CMAKE_BINARY_DIR}/${CMAKE_CFG_INTDIR}/lib${LLVM_LIBDIR_SUFFIX}")
-set(CLANG_RESOURCE_PATH "${LLDB_PATH_TO_SWIFT_BUILD}/lib${LLVM_LIBDIR_SUFFIX}/swift/clang")
+
+if(EXISTS ${LLDB_PATH_TO_SWIFT_BUILD}/lib${LLVM_LIBDIR_SUFFIX}/swift/clang)
+  set(CLANG_RESOURCE_PATH "${LLDB_PATH_TO_SWIFT_BUILD}/lib${LLVM_LIBDIR_SUFFIX}/swift/clang")
+else()
+  set(CLANG_VERSION ${LLVM_VERSION_MAJOR}.${LLVM_VERSION_MINOR}.${LLVM_VERSION_PATCH})
```

Comment:
Clang creates this directory and the name is assembled from `CLANG_VERSION_*` equivalents. In practice, they will be the same usually, but they can be overwritten in principle.

Reply:
Right, however, there is no way to get that value unfortunately.  The `ClangClang.cmake` does not provide the value, so I had to improvise.  There also is no `clang-resource-headers` target available (standalone builds :-( ... I really want to fix swift to enable unified builds with distributions, but that has been a hard battle against @gottesmm).  In the mean time, this is the best that I could come up with.

Reply:
> The ClangClang.cmake does not provide the value
It could be added to `ClangConfig.cmake` but I agree that it's not worth the effort.

> There also is no clang-resource-headers target available
It's still called `clang-headers` here on stable https://reviews.llvm.org/D58791

I think this is acceptable.